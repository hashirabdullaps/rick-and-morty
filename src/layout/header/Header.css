.header__logo {
    cursor: pointer;
    height: 90%;
}

.header {
  background: black !important;
  display: flex;
  justify-content: center;
  align-items: center;
  height: 128px !important;
}
