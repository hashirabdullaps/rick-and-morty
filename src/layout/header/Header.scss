.header__logo {
    cursor: pointer;
    height: 100%;
}

.header {
  display: flex;
  justify-content: center;
  align-items: center;
  position: fixed;
  z-index: 1;
  width: 100%;    
  filter: drop-shadow(2px 4px 6px #001529);
}
