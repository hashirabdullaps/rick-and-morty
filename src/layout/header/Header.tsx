import { Layout } from "antd";
import { useNavigate } from "react-router-dom";

import Logo from "../../assets/images/logo.png";

import "./Header.css";

const Header = () => {
    const navigate = useNavigate();
    return <Layout.Header className="header">
        <img
            src={Logo}
            alt='logo'
            className="header__logo"
            onClick={() => { navigate('/') }}
        />
    </Layout.Header>
};

export default Header;
