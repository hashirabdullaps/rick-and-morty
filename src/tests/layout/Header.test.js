import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../../layout/header/Header';
import { BrowserRouter } from "react-router-dom";

test('should test Header component', () => {
    const component = renderer.create(
        <BrowserRouter>
            <Header />
        </BrowserRouter>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});