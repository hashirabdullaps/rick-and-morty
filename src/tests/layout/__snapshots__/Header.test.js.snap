// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test Header component 1`] = `
<header
  className="ant-layout-header header"
>
  <img
    alt="logo"
    className="header__logo"
    onClick={[Function]}
    src="logo.png"
  />
</header>
`;
